src/components/MoneyRequestConfirmationList.js

```diff
@@ -290,15 +296,24 @@ function MoneyRequestConfirmationList(props) {
             optionHoveredStyle={canModifyParticipants ? styles.hoveredComponentBG : {}}
             footerContent={footerContent}
         >
-            <MenuItemWithTopDescription
-                shouldShowRightIcon={!props.isReadOnly}
-                title={formattedAmount}
-                description={translate('iou.amount')}
-                onPress={() => Navigation.navigate(ROUTES.getMoneyRequestAmountRoute(props.iouType, props.reportID))}
-                style={[styles.moneyRequestMenuItem, styles.mt2]}
-                titleStyle={styles.moneyRequestConfirmationAmount}
-                disabled={didConfirm || props.isReadOnly}
-            />
+            {props.receiptPath && props.isImage && (
```

Comment:
NAB You could nest these conditionals if you wanted, maybe a bit more readable:

```js
{props.receiptPath && (
    props.isImage ? (
        <Image
             style={styles.moneyRequestImage}
             source={{uri: props.receiptPath}}
        />
    ) : (
        <Lounge />
    )
)}
```

Reply:
FYI this was updated for the updated file types, so we were able to simplify it a bit.